docs/fabric-sdk.md

```diff
@@ -10,3 +10,99 @@ It consists of the following layers:
 - `Driver API`: This API translates the `Fabric API` to a concrete driver implementation.
 - `Driver Implementations`: The Fabric SDK comes with a driver implementation for Fabric V2+.
 
+## Configuration
+
+Here is an annotated example of the configuration of the Fabric SDK:
+
+```yaml
+# The fabric section defines the configuration of the fabric networks.  
+fabric:
+  # Is the fabric-sdk enabled
+  enabled: true
+  # The name of the network
+  default:
+    # Is this the default network?
+    default: true
+    # The MSP config path of the default identity to connect to this network.
+    mspConfigPath: /crypto/peerOrganizations/org1.example.com/peers/approver.org1.example.com/msp
+    # Local MSP ID of the default identity
+    localMspId: Org1MSP
+    # Cache size to use when handling idemix pseudonyms. If the value is larger than 0, the cache is enabled and
+    # pseudonyms are generated in batches of the given size to be ready to be used.
+    mspCacheSize: 500
+    msps:
+      - id: idemix # The id of the identity. 
+        mspType: idemix # The type of the MSP.
+        mspID: IdemixOrgMSP # The MSP ID.
+        # The path to the MSP folder containing the cryptographic materials.
+        path: crypto/peerOrganizations/org2.example.com/peers/alice.org2.example.com/extraids/idemix
+    tls:
+      enabled:  true
+      clientAuthRequired: false
+      cert:
+        file: /crypto/peerOrganizations/org1.example.com/peers/approver.org1.example.com/tls/server.crt
+      key:
+        file: /crypto/peerOrganizations/org1.example.com/peers/approver.org1.example.com/tls/server.key
+      clientCert:
+        file: /crypto/peerOrganizations/org1.example.com/peers/approver.org1.example.com/tls/server.crt
+      clientKey:
+        file: /crypto/peerOrganizations/org1.example.com/peers/approver.org1.example.com/tls/server.key
+      rootcert:
+        file: /crypto/peerOrganizations/org1.example.com/peers/approver.org1.example.com/tls/ca.crt
+      clientRootCAs:
+        files:
+          - /crypto/peerOrganizations/org1.example.com/peers/approver.org1.example.com/tls/ca.crt
+      rootCertFile: /crypto/ca-certs.pem
+      keepalive:
+        client:
+          interval: 60s
+          timeout: 600s
+        server:
+          interval: 60s
+          timeout: 600s
+          minInterval: 60s
+    # This section is dedicated to interactions with the ordering service
+    broadcast:
```

Comment:
as the comment already describes ... why don't we just call it `ordering` as this is the fabric term?